scripts/biomedical/VirtualMetabolicHuman/format_microbes.py

```diff
@@ -0,0 +1,90 @@
+'''
+Author: Suhana Bedi
+Date: 07/10/2021
+Name: format_microbe.py
+Description: Add dcids for all the microbes present in the data base, and query
+using the datacommons to see if a node the microbe already exists.
+Declare the columns, 'oxygenstat', 'metabolism', 'gram', 'mtype' and 'platform'
+as enums
+@file_input: input .tsv from VMH with microbiome data
+@file_output: csv output file with enums and dcids generated
+'''
+import sys
+import pandas as pd
+import datacommons as dc
+
+def main():
+    file_input = sys.argv[1]
```

Comment:
Similarly, this main function can be broken up into helper functions.

Reply:
Done !